### ISSUE TITLE
Anti-IAA

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#7242]
**Preferred term label:**
Anti-insulin antibody positivity

**Synonyms**
Anti-IAA antibody positivity

**Definition (free text, please give PubMed ID)**
PMID: 17143785

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:103919

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by MaudDLVT** [comment]
To merge with HP:5000095

**Comment by pnrobinson** [comment]
Adding term as HP:034062. There is no HP:5000095, and I do not think we had a synonym previously.
@MaudDLVT  please reopen if there was a previous anti-insulin antibody term?

**Comment by MaudDLVT** [comment]
this is linked to the issue #7044
<https://github.com/obophenotype/human-phenotype-ontology/issues/7044>.
Thanks,
Maud

Le sam. 18 déc. 2021 à 15:12, Peter Robinson ***@***.***> a
écrit :
